cmd/run.go

```diff
@@ -540,6 +540,16 @@ func executeRun(runTemplateName, runFilePath string, apps []runfileconfig.App) (
 		// Update extended metadata with run file path.
 		putRunTemplateNameInMeta(runState, runTemplateName)
 
+		// Update extended metadata with app log file path.
+		if app.AppLogDestination != "console" {
```

Comment:
I think there is a constant for `console`, please use that.

Reply:
Done.